stylelint-polaris/tests/fixture.css

```diff
@@ -0,0 +1,53 @@
+:root {
+  --pc-omg: orange;
+  --junk: brown;
```

Comment:
Just so I'm clear this text fixture outlines failing rules in a Polaris consumer project? I ask because `--junk` would be valid in a consumers `.scss` file.

Reply:
I don't mind if it's a polaris consumer or polaris-react project. I think we just need a baseline test to get started.